example/.babelrc

```diff
@@ -47,7 +48,8 @@
             "loose": true,
             "useBuiltIns": true
         }],
-        "react"
+        "react",
+        "es2015"
```

Comment:
@doctyper @jamsea The `es2015` preset is needed so that node, which Jest runs in, understands `import`. It was not needed previously because karma utilized webpack to bundle the source and webpack handled the `import`.